### PR TITLE
bigtable: add region tags

### PR DESCRIPTION
**Pull request by billyjacobson**
Adding region tags for Cloud Bigtable samples

### CODE REVIEW

File: bigtable/helloworld/main.go [hunk 1]
```diff
@@ -86,14 +91,18 @@ func main() {
 			log.Fatalf("Could not create column family %s: %v", columnFamilyName, err)
 		}
 	}
+	// [END bigtable_hw_create_table]
 
 	// Set up Bigtable data operations client.
 	// NewClient uses Application Default Credentials to authenticate.
+	// [START bigtable_hw_connect_data]
```
Comment (broady) [inline bigtable/helloworld/main.go]: Do you not want the preceding comments? Seems useful!

File: bigtable/helloworld/main.go [hunk 2]
```diff
@@ -59,11 +61,14 @@ func main() {
 
 	// Set up admin client, tables, and column families.
 	// NewAdminClient uses Application Default Credentials to authenticate.
+	// [START bigtable_hw_connect]
```
Comment (broady) [inline bigtable/helloworld/main.go]: Same here.

File: bigtable/helloworld/main.go (hunk 1 above)
Comment (billyjacobson) [reply bigtable/helloworld/main.go]: The docs already have comments about it, so it seems easier for them to be maintained there. Same for below. Thoughts? 

https://cloud.google.com/bigtable/docs/samples-go-hello

File: bigtable/helloworld/main.go [hunk 3]
```diff
@@ -24,6 +25,7 @@ import (
 
 	"cloud.google.com/go/bigtable"
 )
+// [END bigtable_hw_imports]
```
Comment (broady) [inline bigtable/helloworld/main.go]: Need to add a newline before this to make `gofmt` happy.

File: bigtable/helloworld/main.go (hunk 1 above)
Comment (broady) [reply bigtable/helloworld/main.go]: SGTM!

**Pull request merged**